web/beacon-app/src/utils/getInitials.ts

```diff
@@ -1,13 +1,9 @@
-/**
- * Get initials of a given name (e.g: Calumn Scott -> CS)
- * @param name
- * @returns string
- */
-export default function getInitials(name = '') {
-  return name
-    .match(/(^\S\S?|\b\S)?/g)
-    ?.join('')
-    .match(/(^\S|\S$)?/g)
-    ?.join('')
-    .toUpperCase();
-}
+export const getInitials = (name: string) => {
+  const nameArray = name?.split(' ') || [''];
```

Comment:
While toggling between onboarded and not onboarded user states, I kept getting an undefined error related to this function. Using optional chaining  `(? after name)`  didn't completely resolve it, so `['']` was added as a fallback.

Reply:
the updated function is easier to follow as well - thanks for making this fix!